bundled/muparser_v2_3_3/src/muParserTest.cpp

```diff
@@ -1546,17 +1546,7 @@ namespace mu
 					// The tests equations never result in infinity, if they do thats a bug.
 					// reference:
 					// http://sourceforge.net/projects/muparser/forums/forum/462843/topic/5037825
-#ifdef _MSC_VER
-#pragma warning(push)
-#pragma warning(disable:4127)
-#endif
-					if (std::numeric_limits<value_type>::has_infinity)
-#ifdef _MSC_VER
-#pragma warning(pop)
-#endif
-					{
-						bCloseEnough &= (fabs(fVal[i]) != numeric_limits<value_type>::infinity());
-					}
+					bCloseEnough &= std::isfinite(fabs(fVal[i]));
```

Comment:
Nice work.

Do we still need `fabs()` here?

Reply:
Probably not :-) I will change that line in both patches.